### PR TITLE
os: use kmm APIs

### PR DESCRIPTION
**Pull request by sunghan-chang**
Kernel modules should allocate the memory in kernel space for internal usages.
Let's use kmm APIs instead of memory allocation APIs for user.
This commit includes the change in arch, board, crypto, drivers and logm.

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

**Comment by sunghan-chang** [comment]
NEED TO VERIFY

File: os/arch/xtensa/src/esp32/spi_flash/flash_mmap.c [hunk 1]
```diff
@@ -145,7 +145,7 @@ esp_err_t IRAM_ATTR spi_flash_mmap(size_t src_addr, size_t size, spi_flash_mmap_
 #ifdef CONFIG_SPIRAM_SUPPORT
 	int *pages = heap_caps_malloc(sizeof(int) * page_count, MALLOC_CAP_INTERNAL);
```
Comment (jeongchanKim) [inline os/arch/xtensa/src/esp32/spi_flash/flash_mmap.c:146]: In `heap_caps_malloc`, it still allocates the memory through `malloc_at`. We'd better change this also.

Comment (sunghan-chang) [reply os/arch/xtensa/src/esp32/spi_flash/flash_mmap.c:146]: Done

**Review by jeongchanKim — APPROVED** [review]
Looks good to me.

**Review by jeongarmy — APPROVED** [review]
LGTM

**Review by Taejun-Kwon — APPROVED** [review]

**Pull request merged**